index.js

```diff
@@ -391,6 +391,37 @@
       [$.Function, b, a, c],
       function(f, x, y) { return f(y, x); });
 
+  //# map :: Functor f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a function and a [Functor][], applies the function to each of the
+  //. functor's values, and returns a functor of the same shape.
+  //.
+  //. ```javascript
+  //. > S.map(S.mult(4), S.Just(2))
+  //. S.Just(8)
+  //.
+  //. > S.map(S.mult(4), S.Nothing())
+  //. S.Nothing()
+  //.
+  //. > S.map(S.mult(2), [1, 2, 3])
+  //. [2, 4, 6]
+  //. ```
+  S.map =
```

Comment:
Let's write `var map = S.map =` and use `map` (rather than `S.map`) elsewhere in the file.